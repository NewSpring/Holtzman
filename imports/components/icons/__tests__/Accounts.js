import renderer from "react-test-renderer";
import AccountIcons from "../Accounts";

describe("Account Icons", () => {
  describe("AmEx Icon", () => {
    it("should exist", () => {
      const amEx = renderer.create(
        <AccountIcons.AmEx />
      );
      expect(amEx).toBeDefined();
    });

    it("should match snapshot", () => {
      const amEx = renderer.create(
        <AccountIcons.AmEx />
      );
      expect(amEx).toMatchSnapshot();
    });
  });

  describe("Visa Icon", () => {
    it("should exist", () => {
      const visa = renderer.create(
        <AccountIcons.Visa />
      );
      expect(visa).toBeDefined();
    });

    it("should match snapshot", () => {
      const visa = renderer.create(
        <AccountIcons.Visa />
      );
      expect(visa).toMatchSnapshot();
    });
  });

  describe("Discover Icon", () => {
    it("should exist", () => {
      const discover = renderer.create(
        <AccountIcons.Discover />
      );
      expect(discover).toBeDefined();
    });

    it("should match snapshot", () => {
      const discover = renderer.create(
        <AccountIcons.Discover />
      );
      expect(discover).toMatchSnapshot();
    });
  });

  describe("MasterCard Icon", () => {
    it("should exist", () => {
      const masterCard = renderer.create(
        <AccountIcons.MasterCard />
      );
      expect(masterCard).toBeDefined();
    });

    it("should match snapshot", () => {
      const masterCard = renderer.create(
        <AccountIcons.MasterCard />
      );
      expect(masterCard).toMatchSnapshot();
    });
  });

  describe("Bank Icon", () => {
    it("should exist", () => {
      const bank = renderer.create(
        <AccountIcons.Bank />
      );
      expect(bank).toBeDefined();
    });

    it("should match snapshot", () => {
      const bank = renderer.create(
        <AccountIcons.Bank />
      );
      expect(bank).toMatchSnapshot();
    });
  });
});
