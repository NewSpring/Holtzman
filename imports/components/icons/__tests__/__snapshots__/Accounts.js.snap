exports[`Account Icons AmEx Icon should match snapshot 1`] = `
<div
  className="background--fill display-inline-block geSTCm"
  height={undefined}
  width={undefined} />
`;

exports[`Account Icons Bank Icon should match snapshot 1`] = `
<div
  className="background--fill display-inline-block hTUpnQ"
  height={undefined}
  width={undefined} />
`;

exports[`Account Icons Discover Icon should match snapshot 1`] = `
<div
  className="background--fill display-inline-block jwnrVx"
  height={undefined}
  width={undefined} />
`;

exports[`Account Icons MasterCard Icon should match snapshot 1`] = `
<div
  className="background--fill display-inline-block OVyYV"
  height={undefined}
  width={undefined} />
`;

exports[`Account Icons Visa Icon should match snapshot 1`] = `
<div
  className="background--fill display-inline-block kAaGle"
  height={undefined}
  width={undefined} />
`;
